pyproject.toml

```diff
@@ -19,17 +19,17 @@ license = "Apache-2.0"
 name = "pyprql"
 readme = "README.md"
 repository = "https://github.com/prql/pyprql"
-version = "0.10.0"
+version = "0.11.0"
```

Comment:
Do not manually increase the version. The version should be determined automatically by CI.

Reply:
Thanks, corrected